### ISSUE TITLE
introduce an org-unit as (non-unique) identifier

### ISSUE THREAD

**Issue by ctr49** [netzbegruenung/green-directory#59]
A certain organization (i.e. an "Ortsverband") may span multiple cities, so it may make sense to introduce an org-unit or similar attribute that clearly identifies the legal entity while it would be possible to have multiple entries (i.e. with different cities) belonging to that org-unit.
For entries that don't have an org-unit we could assume that city = org-unit.

**Comment by marians** [comment]
Can you offer an example to illustrate this?

**Comment by ctr49** [comment]
Yes: an "Ortsverband" may span multiple cities. When each city within the Ortsverband has an own homepage then we get those strange entries like https://github.com/netzbegruenung/green-directory/commit/c309bcb58e9acf3fbcd2372957f8467693ebe720

Instead I'd rather have a schema like:

type: REGIONAL_CHAPTER
level: DE:ORTSVERBAND
country: DE
state: Bundesland
district: Kreisverband
city: Stadt
ou: Ortsverband-Name
urls:
  - type: WEBSITE
    url: https://www.just-a-single-web-page-per-entry.de


Where ou is absent (which would be any existing entries, ou can be assumed to be the city (this is more usual anyway)

**Comment by ctr49** [comment]
Update: I think the current assumption that an ORTSVERBAND only spans a single city is not valid.

**Comment by marians** [comment]
Indeed. I just had to add this entry for an OV covering three cities: https://github.com/netzbegruenung/green-directory/pull/90/commits/2c0ae02cc9c1de633051e1a3d9b64ee588599220

The first problem I have with that is that our current `city` attribute only allows for one value, and the value in the above commit is a city that does not exist. As a solution, I'd like to have `city` accept multiple values. Example:

```yaml
type: REGIONAL_CHAPTER
level: DE:ORTSVERBAND
country: DE
state: Brandenburg
district: Dahme-Spreewald
city:
  - Eichenwalde
  - Schulzendorf
  - Zeuthen
urls:
  - type: WEBSITE
    url: http://gruene-in-zes.blogspot.com/
```

Another problem is that there is no attribute to determine the actual name of the entity. That's what the `ou` atttribute as you propose it would solve. However why not have it more self-explanatory and universally usable, like `name`?:

```yaml
name: Zeuthen, Eichenwalde & Schulzendorf
```

**Comment by hejamu** [comment]
The `name` attribute would be very important for some "Ortsverbände" because some don't correspond to the name of the city oder cities they act in. See #213 .

**Comment by ctr49** [comment]
Yes, enabling city to be a list makes sense as well as using "name" instead of ou.
To avoid changing all entries my suggestions stand to assume that city = name if there is no name and city just has one entry.
